# HG changeset patch
# User andrew
# Date 1526489197 -3600
#      Wed May 16 17:46:37 2018 +0100
# Node ID 64e87a408afd2b56d59dad73dee28d4b99463810
# Parent  00ccc73498628a51a45301322e64ce2ad06e49be
PR3591: Fix for bug 3533 doesn't add -mstackrealign to JDK code

diff --git openjdk.orig///common/autoconf/generated-configure.sh openjdk///common/autoconf/generated-configure.sh
--- openjdk.orig///common/autoconf/generated-configure.sh
+++ openjdk///common/autoconf/generated-configure.sh
@@ -4000,7 +4000,7 @@
 #CUSTOM_AUTOCONF_INCLUDE
 
 # Do not change or remove the following line, it is needed for consistency checks:
-DATE_WHEN_GENERATED=1526334033
+DATE_WHEN_GENERATED=1526489141
 
 ###############################################################################
 #
@@ -28933,6 +28933,8 @@
 
   fi
 
+    CFLAGS_JDK="${CFLAGS_JDK} ${REALIGN_CFLAG}"
+    CXXFLAGS_JDK="${CXXFLAGS_JDK} ${REALIGN_CFLAG}"
 
   fi
 
diff --git openjdk.orig///common/autoconf/toolchain.m4 openjdk///common/autoconf/toolchain.m4
--- openjdk.orig///common/autoconf/toolchain.m4
+++ openjdk///common/autoconf/toolchain.m4
@@ -794,6 +794,8 @@
     TOOLCHAIN_COMPILER_CHECK_ARGUMENTS([$REALIGN_CFLAG -Werror], [],
       AC_MSG_ERROR([The selected compiler $CXX does not support -mstackrealign! Try to put another compiler in the path.])
     )
+    CFLAGS_JDK="${CFLAGS_JDK} ${REALIGN_CFLAG}"
+    CXXFLAGS_JDK="${CXXFLAGS_JDK} ${REALIGN_CFLAG}"
     AC_SUBST([REALIGN_CFLAG])
   fi
 
